src/arch/arm/common_arm.lds

```diff
@@ -64,7 +64,7 @@ SECTIONS
         *(.bss)
         *(COMMON) /* fallback in case '-fno-common' is not used */
 
-        /* 4k breakpoint stack */
+        /* 4k breakpoint stack, needed for PMODE_ABORT on ARM */
```

Comment:
Seems it's used on AARCH32 only, not on AARCH64. Might be worth documenting at least, not sure if there is a simple way to remove it on AARCH64 also.